addon/routes/dashboard.js

```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default class DashboardRoute extends Route {
+  beforeModel() {
+    this.transitionTo('dashboard.week');
```

Comment:
`transitionTo` on the route is deprecated. Need to import the router service see https://github.com/ilios/common/pull/2688/commits/0db3bea18c8428ee78418c9502b746dc706e291f